.gitattributes

```diff
@@ -7,3 +7,7 @@
 # Have Github ignore js vendored files.
 # https://github.com/gramps-project/gramps/tree/master/data/javascript
 #data/javascript/*.js linguist-vendored
+
+# don't mess with line endings for Gedcom files
+*.ged binary

```

Comment:
Git still does diff check with binary mode, it just doesn't show lines. (I tested this carefully).  As near as I can tell, leaving '*.ged' out of .gitattributes, leaves git to decide on its own that this is text and to do EOL changes, and possibly trailing space removal, either of which messes up the Gedcoms.